ui/components/app/flask/snaps-authorship-pill/snaps-authorship-pill.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Chip from '../../../ui/chip';
+import Box from '../../../ui/box';
+import Typography from '../../../ui/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+} from '../../../../helpers/constants/design-system';
+
+const SnapsAuthorshipPill = ({ packageName, className, url }) => {
+  return (
+    <a href={url}>
+      <Chip
+        leftIcon={
+          <Box paddingLeft={2}>
+            <i className="fab fa-npm fa-lg snaps-authorship-icon"></i>
```

Comment:
Can you short hand to close tag here
```suggestion
            <i className="fab fa-npm fa-lg snaps-authorship-icon" />
```

Reply:
Yup